packages/runtime-vapor/src/apiCreateSlots.ts

```diff
@@ -0,0 +1,61 @@
+// NOTE: this filed is based on `runtime-core/src/helpers/createSlots.ts`
+
+import { isArray } from '@vue/shared'
+import type { InternalSlots, Slot } from './componentSlots'
+import { renderEffect } from './renderEffect'
+
+// TODO: SSR
+
+interface CompiledSlotDescriptor {
+  name: string
+  fn: Slot
+  key?: string
+}
+
+export const createSlots = (
```

Comment:
Let's put this function into `createComponent` instead, because `createSlots` needs sub component instance.

Reply:
And move this file to `packages/runtime-vapor/src/componentSlots.ts`